osu.Game/Screens/OnlinePlay/Playlists/PlaylistsReadyButton.cs

```diff
@@ -80,9 +86,16 @@ public override LocalisableString TooltipText
             }
         }
 
-        private bool enoughTimeLeft =>
+        private bool enoughTimeLeft()
+        {
+            // this doesn't consider mods which apply variable rates, yet.
```

Comment:
The code to resolve/apply the rate with mods I had copied from somewhere else, including this comment. However, I assume this comment should be somewhere inside `CalculateRateWithMods` instead?

Reply:
Yes, please move the TODO to that metho.d